playbooks/amc.yml

```diff
@@ -173,22 +173,30 @@
             ignored-fs-types: "^(sys|proc|auto|shm|tmp|ns)fs$"
     - { role: deploy_info }
 
-# TODO: Fix the notifier. It's breaking the Ansible deployments. See RED-133
-#- name: Install xqueue and notifier on services node
-#  hosts: "services"
-#  become: True
-#  become_method: sudo
-#  gather_facts: True
-#  vars:
-#    migrate_db: "yes"
-#  roles:
-#    - mysql_init
-#    - edxapp_common
-#    - role: nginx
-#      nginx_sites:
-#      - xqueue
-#    - { role: notifier, NOTIFIER_DIGEST_TASK_INTERVAL: "5" }
-#    - { role: "xqueue", update_users: True }
-#    - role: custom_domains # This gets the list of domains
-#      when: nginx_enable_custom_domains|default(False)
-#    - { role: letsencrypt_role, letsencrypt_webserver: "" }
+
+- name: Install xqueue and notifier on services node
+  hosts: "services"
+  become: True
+  become_method: sudo
+  gather_facts: True
+  vars:
+    migrate_db: "yes"
+  roles:
+    - role: mysql_init
+      when: TEMP_DEBUG_XQUEUE|default(false)
+    - role: edxapp_common
+      when: TEMP_DEBUG_XQUEUE|default(false)
+    - role: nginx
+      nginx_sites:
+      - xqueue
+      when: TEMP_DEBUG_XQUEUE|default(false)
+    # TODO: Fix the notifier. It's breaking the Ansible deployments. See RED-133
+    #- { role: notifier, NOTIFIER_DIGEST_TASK_INTERVAL: "5" }
+    - role: "xqueue"
+      update_users: True
+      when: TEMP_DEBUG_XQUEUE|default(false)
+    - role: custom_domains # This gets the list of domains
+      when: nginx_enable_custom_domains|default(False)
+    - role: letsencrypt_role
+      letsencrypt_webserver: ""
+      when: nginx_enable_custom_domains|default(False)
```

Comment:
There's a good chance that this is not needed, or will fail when installed. I think those steps never ran on Hawthorn because Notifier would fail.

```suggestion
```

Reply:
@OmarIthawi Good point! @thraxil Can you confirm?

Reply:
Maybe? I honestly have no idea what Xqueue is, why it had anything to do with custom domains in the past, or whether it will break now. Everything here is happening on the `services` node, which the load balancer isn't routing any external traffic to and doesn't have the certs bucket mounted, so it seems like it ought to be completely independent of custom domains stuff.

Reply:
I read a doc about xQueue [here](https://openedx.atlassian.net/wiki/spaces/AC/pages/94307018/XQueue). 

- Under **Future state / wish list**: `Decrease coupling: pull the LMS XQueue interface out of CAPA, or at least remove the code dependency between Certificates and CAPA.`

Is this why we need both xQueue and CAPA to handle both instructor and student generated certificates?